src/schema/v2/user.ts

```diff
@@ -169,6 +173,72 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
         }
       },
     },
+    follows: {
+      type: new GraphQLObjectType({
+        name: "UserFollows",
+        fields: {
+          artistFollowsConnection: {
+            type: artistConnection.connectionType,
+            args: pageable({}),
+            resolve: async ({ id }, args, { userArtistFollowsLoader }) => {
+              if (!userArtistFollowsLoader) {
+                throw new Error(
+                  "Loader not found. You must supply an X-Access-Token header."
+                )
+              }
+              const { page, size, offset } = convertConnectionArgsToGravityArgs(
+                args
+              )
+
+              const { body, headers } = await userArtistFollowsLoader(id, {
```

Comment:
Should there be a meaningful default `sort` passed-in (like most recent)? It doesn't look like the Gravity endpoint currently supports that, perhaps that's a reasonable follow-up (to add to Gravity and here).

Reply:
Yeah, was just thinking this while working on the client side! I'll handle sorting there, for now, and ticket out a follow up.